tests/test_misc.py

```diff
@@ -1908,6 +1909,113 @@ def test_spends_conflicting(revault_network, bitcoind):
     )
 
 
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_spend_threshold(revault_network, bitcoind, executor):
+    CSV = 2016
```

Comment:
nit: we don't really need a CSV that big, it'll just slow down the test by mining blocks